openproblems/tasks/_batch_integration/batch_integration_graph/datasets/lung.py

```diff
@@ -0,0 +1,36 @@
+from .....data.lung import load_lung
+from .....tools.decorators import dataset
+
+
+@dataset(
+    dataset_name="Lung (by batch)",
+    data_url=load_lung.metadata["data_url"],
+    data_reference=load_lung.metadata["data_reference"],
+    dataset_summary="Human pancreatic islet scRNA-seq data from 6 datasets "
+    "across technologies (CEL-seq, CEL-seq2, Smart-seq2, inDrop, Fluidigm C1, "
+    "and SMARTER-seq).",
```

Comment:
This is definitely the wrong summary

Reply:
When adding the summary, please add: number of batches, cells, and donors/samples